packages/app-store/_utils/useIsAppEnabled.ts

```diff
@@ -0,0 +1,62 @@
+import { useState, useEffect } from "react";
+import type { ZodType } from "zod";
+
+import type { SetAppDataGeneric, GetAppDataGeneric } from "../EventTypeAppContext";
+import type { EventTypeAppCardApp } from "../types";
+
+function useIsAppEnabled<TAppData extends ZodType>(
+  app: EventTypeAppCardApp,
+  getAppData: GetAppDataGeneric<TAppData>,
+  setAppData: SetAppDataGeneric<TAppData>
```

Comment:
I think instead of passing these as args we should be able to get them from `useAppContextWithSchema`